recipes/cargo-make/conda_build_config.yaml

```diff
@@ -0,0 +1,11 @@
+# https://conda-forge.org/docs/maintainer/knowledge_base.html#requiring-newer-macos-sdks
+MACOSX_SDK_VERSION:            # [osx and x86_64]
+  - '10.12'                    # [osx and x86_64]
+MACOSX_DEPLOYMENT_TARGET:      # [osx and x86_64]
+  - '10.12'                    # [osx and x86_64]
+
+channel_targets:
+  - conda-forge rust_dev       # [win]
```

Comment:
This would otherwise upload it to `rust_dev`. This is probably not what you intended. Probably meant `channel_sources`? This has a slightly different syntax.
```suggestion
```

Reply:
I probably only need the arch definition. Giving it a try